### ISSUE TITLE
k90r not actually TF2T?

### ISSUE THREAD

**Issue by marcharper** [Axelrod-Python/axelrod-fortran#83]
While updating this library for random seeding I've found that the following code doesn't behave as expected:

```
import axelrod as axl
import axelrod_fortran as axlf

player1 = axlf.Player("k90r")
player2 = axl.Defector()

match = axl.Match((player1, player2), turns=5)
results = match.play()
print(results)
```

It prints
```
[(C, D), (C, D), (C, D), (C, D), (C, D)]
```

However `player1` should have defected after two opponent defections, i.e. on round 3.

The test `test_implemented_strategies` in `test_player.py` fails when I run it directly for `k90r` and TF2T.

Note that TF2T in axelrod does defect on the third round:

```
import axelrod as axl

player1 = axl.TitFor2Tats()
player2 = axl.Defector()

match = axl.Match((player1, player2), turns=5)
results = match.play()
print(results)
```

```
[(C, D), (C, D), (D, D), (D, D), (D, D)]
```

**Comment by marcharper** [comment]
Here's the tour exec code for [k90r](https://github.com/Axelrod-Python/TourExec/blob/master/src/strategies/k90r.f). It looks like TF2T...

**Comment by meatballs** [comment]
Looks like we have a rogue space on L10

**Comment by marcharper** [comment]
Looks like a regression of #59 ? Or perhaps my local install of TourExec is old?

**Comment by meatballs** [comment]
Should be fixed in v0.4.1 of TourExec

**Comment by marcharper** [comment]
It was an old install. Closing.
